components/LinkButton.tsx

```diff
@@ -6,12 +6,21 @@ export interface LinkButtonProps {
   text: string
   id?: string
   lang?: string
+  external?: boolean
 }
 
-const LinkButton: FC<LinkButtonProps> = ({ href, text, id, lang }) => {
+const LinkButton: FC<LinkButtonProps> = ({
+  href,
+  text,
+  id,
+  lang,
+  external,
+}) => {
   return (
     <Link href={href} passHref>
       <a
+        target={external ? '_blank' : ''}
+        rel={external ? 'noopener noreferrer' : ''}
```

Comment:
Could use && here

Reply:
Will be switching to using the LinkSummary component but I'll still update this for external links if for whatever reason we want to link to an external site via a button in the future.